content/en/docs/tutorials/stateless-application/guestbook.md

```diff
@@ -1,28 +1,24 @@
 ---
-title: "Example: Deploying PHP Guestbook application with Redis"
+title: "Example: Deploying PHP Guestbook application with MongoDB"
 reviewers:
 - ahmetb
 content_type: tutorial
 weight: 20
 card:
   name: tutorials
   weight: 30
-  title: "Stateless Example: PHP Guestbook with Redis"
+  title: "Stateless Example: PHP Guestbook with MongoDB"
 ---
 
 <!-- overview -->
 This tutorial shows you how to build and deploy a simple, multi-tier web application using Kubernetes and [Docker](https://www.docker.com/). This example consists of the following components:
 
-* A single-instance [Redis](https://redis.io/) master to store guestbook entries
-* Multiple [replicated Redis](https://redis.io/topics/replication) instances to serve reads
+* A single-instance [MongoDB](https://www.mongodb.com/) to store guestbook entries
 * Multiple web frontend instances
```

Comment:
I recommend explaining that a single-instance MongoDB cluster is a bad idea, and the reason for the simplicity is to focus on the stateless web app component.

Reply:
added a non-production readiness statement a few lines up